.openpublishing.publish.config.json

```diff
@@ -76,7 +76,8 @@
       "monikers": [],
       "moniker_ranges": [
         "vsmac-2017",
-        ">=vsmac-2019"
+        "vsmac-2019",
+        ">=vsmac-2022"
```

Comment:
I don't actually know what this does. But I've noticed that when I go to https://docs.microsoft.com/en-us/visualstudio/mac/installation, it picks the vsmac-2019 moniker instead of vsmac-2022, and I don't know how to change that.

Reply:
Hi @sandyarmstrong 

Making these changes in the private repo would enable staging and validation. That way you could test your changes and see if they work/tweak things as needed.